### PR TITLE
Added README.md

### PR DESCRIPTION
**Pull request by ykate1998**
Added README for global features model

### CODE REVIEW

**Review by andrefaraujo — CHANGES_REQUESTED** [review]
Thanks Kate, this looks great. I found one small nit, see comment.

Another comment I had is: once the model is trained, how can we export the model to evaluate in a different dataset? Eg, something like this for DELG: https://github.com/tensorflow/models/tree/master/research/delf/delf/python/training#exporting-the-trained-model

**Comment by andrefaraujo** [comment]
@ykate1998 friendly ping :)

File: research/delf/delf/python/training/model/export_CNN_global.py [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+    named_output_tensors['global_descriptors'] = tf.identity(
+      extracted_features, name='global_descriptors')
+    return named_output_tensors
+
+  @tf.function(input_signature=[
+    tf.TensorSpec(shape=[None, None, 3], dtype=tf.uint8, name='input_image')
+  ])
+  def ExtractFeaturesFixedScales(self, input_image):
+    exp_image = tf.expand_dims(input_image, axis=0)
+    return self._model(exp_image, training=False)
```
[the remark this answers is not in the document]
Comment (ykate1998) [reply research/delf/delf/python/training/model/export_CNN_global.py:226]: Thank you for pointing this out

**Review by andrefaraujo — APPROVED** [review]
Looks perfect, thanks!

**Pull request merged**